changelog/unreleased/4387

```diff
@@ -0,0 +1,7 @@
+Enhancement: A description in the UI elements added to improve the accessibility
+
+In the UI elements a description of the meaning or action associated with that element has been included to make the application more accessible.
+Changes improved: toolbar, file list, share, drawer Menu, manage accounts and preview image.
```

Comment:
```suggestion
Enhancement: Content description in UI elements to improve accessibility

A description of the meaning or action associated with some UI elements has been included as alternative text to make the application more accessible.
Views improved: toolbar, file list, spaces list, share, drawer menu, manage accounts and image preview.
```

You forget essential things here, try to explain it better

Reply:
Changed